pandas_datareader/tests/test_tsp.py

```diff
@@ -0,0 +1,16 @@
+import nose
+import pandas.util.testing as tm
+import datetime as dt
+
+import pandas_datareader.tsp as tsp
+
+class TestTSPFunds(tm.TestCase):
+    def test_get_allfunds(self):
+        tspdata = tsp.TSPReader(start='2015-11-2',end='2015-11-2').read()
+        
+        assert len(tspdata == 1)
+        assert round(tspdata['I Fund'][dt.date(2015,11,2)],5) == 25.0058
```

Comment:
fix space after coma

You may probably use `pandas.util.testing.assert_almost_equal`